doc/source/release-notes/Percona-Server-5.7.27-30.rst

```diff
@@ -0,0 +1,128 @@
+.. rn:: 5.7.27-30
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on
+|date|. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.27-30/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+	
+This release is based on `MySQL 5.7.27
+<https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-27.html>`_ and includes
+all the bug fixes in it. |Percona Server| |release| is now the current GA
+(Generally Available) release in the 5.7 series.
+	
+All software developed by Percona is open-source and free.
+
+In this release, |Percona Server| introduces the variable
+:variable:`binlog_skip_flush_commands`. This variable controls whether
+or not ``FLUSH`` commands are written to the binary log. Setting this
+variable to **ON** can help avoid problems in replication. For more
+information, see :ref:`percona-server.binary-log.flush.writing`.
+
+.. note:: 
+
+   If you're currently using |Percona Server| 5.7, Percona recommends upgrading
+   to this version of 5.7 prior to upgrading to |Percona Server| 8.0.
+
+Bugs Fixed
+================================================================================
+
+- Parallel doublewrite buffer writes must crash the server on an I/O error 
+  occurs. Bug fixed :psbug:`5678`.
+
+- On a server with two million or more tables, a shutdown may take a measurable 
+  length of time. Bug fixed :psbug:`5639`.
```

Comment:
upstream bug exists for this one too: https://bugs.mysql.com/bug.php?id=95895

Reply:
I think the description can be improved. This bug is specific to tables having FK and AUTOINC

So something like "On a server with two million or tables using Foriegn Keys and AUTOINC columns, shutdown may take.."

Reply:
Updated.